PowerShellTools/DebugEngine/HostUi.cs

```diff
@@ -157,6 +146,9 @@ private string GetPrompt()
     /// </summary>
     public class HostUi
     {
+        [DllImport("user32.dll")]
+        static extern bool SetForegroundWindow(IntPtr hWnd);

```

Comment:
We start to use this in a few places, including ```SetForegroundWindow(Process.GetCurrentProcess().MainWindowHandle); ``` which is duplicated in a few places.

Did we want to pull these uses and calls out into something more convenient?